### PR TITLE
release 2.17.0

### PR DESCRIPTION
**Pull request by bmc-msft**
Note, this assumes #878, #890, and #898 have been merged.

### CODE REVIEW

**Review by chkeita — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,25 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2.17.0
+### Added
+* Documentation: Added [descriptions](docs/AADEntitites.md) for the Azure AD entities used by OneAzure. [#886](https://github.com/microsoft/onefuzz/pull/896)
```
Comment (ranweiler) [inline CHANGELOG.md:9]:
```suggestion
* Documentation: Added [descriptions](docs/AADEntitites.md) for the Azure AD entities used by OneFuzz. [#896](https://github.com/microsoft/onefuzz/pull/896)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,25 @@ All notable changes to this project will be documented in this file.
...
+* Service: Updated multiple first-party and third-party Python libraries.  [#893](https://github.com/microsoft/onefuzz/pull/893), [#889](https://github.com/microsoft/onefuzz/pull/889), [#866](https://github.com/microsoft/onefuzz/pull/886), [#885](https://github.com/microsoft/onefuzz/pull/885), [#861](https://github.com/microsoft/onefuzz/pull/861), [#890](https://github.com/microsoft/onefuzz/pull/890), 
+* Supervisor: The supervisor now includes the full error context upon failure. [#879](https://github.com/microsoft/onefuzz/pull/879)
+* Service: Cleaned up scaleset update logs. [#880](https://github.com/microsoft/onefuzz/pull/880)
+* Agent: Continued development related to upcoming features. [#874](https://github.com/microsoft/onefuzz/pull/874), [#868](https://github.com/microsoft/onefuzz/pull/868), [#864](https://github.com/microsoft/onefuzz/pull/864)
+* SDK/CLI: Replaced Python based directory uploading with `azcopy sync`.  [#878](https://github.com/microsoft/onefuzz/pull/878)
+
+### Fixed
+* Service/Supervisor: Fixed an issue where scalesets do not shrink or shutdown upon completion of work as expected. [#866](https://github.com/microsoft/onefuzz/pull/866)
```
Comment (ranweiler) [inline CHANGELOG.md:23]:
```suggestion
* Service/Supervisor: Fixed an issue where scalesets do not shrink or shutdown as expected upon completion of work. [#866](https://github.com/microsoft/onefuzz/pull/866)
```

**Review by ranweiler — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (bmc-msft) [reply CHANGELOG.md:23]: This rephrase misses the context.   When I add quotes, does this help?

  "shrink" or "shutdown upon completion of work" as expected.

Comment (ranweiler) [reply CHANGELOG.md:23]: Ah, right! How about:
```suggestion
* Service/Supervisor: Fixed an issue shrinking scalesets where idle nodes would not shut down as expected. [#866](https://github.com/microsoft/onefuzz/pull/866)
```

**Pull request merged**